db/config.cc

```diff
@@ -970,6 +970,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , enable_repair_based_node_ops(this, "enable_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, true, "Set true to use enable repair based node operations instead of streaming based.")
     , allowed_repair_based_node_ops(this, "allowed_repair_based_node_ops", liveness::LiveUpdate, value_status::Used, "replace,removenode,rebuild,bootstrap,decommission", "A comma separated list of node operations which are allowed to enable repair based node operations. The operations can be bootstrap, replace, removenode, decommission and rebuild.")
     , enable_compacting_data_for_streaming_and_repair(this, "enable_compacting_data_for_streaming_and_repair", liveness::LiveUpdate, value_status::Used, true, "Enable the compacting reader, which compacts the data for streaming and repair (load'n'stream included) before sending it to, or synchronizing it with peers. Can reduce the amount of data to be processed by removing dead data, but adds CPU overhead.")
+    , enable_tombstone_gc_for_streaming_and_repair_compaction(this, "enable_tombstone_gc_for_streaming_and_repair_compaction", liveness::LiveUpdate, value_status::Used, false,
+            "Enable compaction during streaming and repair (see enable_compacting_data_for_streaming_and_repair) to garbage-collect tombstones")
```

Comment:
I think that such an esoteric flag requires a whole paragraph to explain it, like you did in this cover letter - this one-sentence explanation won't mean anything to anyone a year from now. Perhaps it deserves a long explanation in https://opensource.docs.scylladb.com/stable/reference/configuration-parameters.html or some repair-related document.